mvp.feature

```diff
@@ -1117,41 +1162,39 @@ Feature: MVP features for NVA
 
   # Actions from Page : My Worklist
   @1252
-  Scenario Outline: Curator expands an item in the Worklist
-    Given that a Curator views My Worklist tabs
+  Scenario: Curator expands an item in the Worklist
+    Given that a Curator views DOI Request tab (Part II)
     When they click Expand on an item
     Then they see the item is expanded
     And they see the Title of the Publication
-    And they see the Message from the User
+    And they see the DOI Request status
+    And they see a list of Messages to and from the Submitter with fields:
+      | Message   |
+      | Submitter |
+      | Date      |
+    And they see an input field for Answer
     And they see buttons
       | Send Answer       |
       | Go to Publication |
       | Archive           |
 
   @358
-  Scenario Outline: Curator opens an item in the Worklist
-    Given that the Curator expands an item in the Worklist
-    And they select a <Tab>
-    When they click Go to Publication on an item
+  Scenario: Curator opens a DOI request
+    Given that a Curator expands an item in the Worklist
+    And the item is a DOI request
+    And they read the optional message from the Owner
```

Comment:
Isn't the word `optional ` an implicit fork of the scenario?

Reply:
removed this line as it is not necessary info for the scenario